realsense2_camera/test/utils/pytest_live_camera_utils.py

```diff
@@ -72,27 +72,29 @@ def get_depth_profiles(long_data, start_index, end_index):
         if len(long_data[line_no]) == 0:
             break
         debug_print("depth profile processing:" + long_data[line_no])
-        depth_profile = long_data[line_no].split()
-        if len(depth_profile) == 5:
-            profile = depth_profile[0]
-            value = depth_profile[1]+"x"+depth_profile[3]
-            img_format = depth_profile[4]
-        elif len(depth_profile) == 6:
-            profile = depth_profile[0]+depth_profile[1]
-            value = depth_profile[2]+"x"+depth_profile[4]
-            img_format = depth_profile[5]
-        elif len(depth_profile) == 7:
-            profile = depth_profile[1]
-            value = depth_profile[2]+"x"+depth_profile[5]
-            img_format = depth_profile[3]
-        elif len(depth_profile) == 8:
-            profile = depth_profile[1]+depth_profile[2]
-            value = depth_profile[3]+"x"+depth_profile[6]
-            img_format = depth_profile[4]
+        enumerate_devices_line_splitted = long_data[line_no].split()
+        if len(enumerate_devices_line_splitted) == 7:
```

Comment:
5 & 6 are not needed anymore?

Reply:
Since We moved the tool output from default to verbose, We don't need 5 and 6.
verbose outputs required data in either spllited length of 7/8.